v20.1/time.md

```diff
@@ -1,39 +1,68 @@
 ---
-title: TIME
+title: TIME / TIMETZ
 summary: The TIME data type stores a time of day in UTC.
 toc: true
 ---
+
 The `TIME` [data type](data-types.html) stores the time of day in UTC.
 
+The `TIMETZ` data type stores a time of day with an optional time zone offset from UTC.
```

Comment:
The phrasing "with an optional time zone offset" is confusing to me. If I don't use the option, it would work just like `TIME` right? So when should I use the option and when should I not? Or is it that `TIMETZ` is the option for `TIME`?

Reply:
Potentially naive question: Which time zone will `TIMETZ` use?

Reply:
if you don't use it, it will default to the timezone of the session (i.e. UTC by default, `SET TIME ZONE ''` if set).